ISSUE_TEMPLATE/testing.md

```diff
@@ -0,0 +1,82 @@
+# Prerequisites
+
+You will need:
+- access to the official CentOS CI fedora-coreos namespace
+- access to the AWS S3 fcos-builds bucket
+- the following packages: `awscli gnupg2 git`
+- [`fedora-coreos-stream-generator`](https://github.com/coreos/fedora-coreos-stream-generator/)
+- your GPG key linked to your FAS account
+
+# Promote testing-devel changes
+
+If you haven't already, check out both [`fedora-coreos-releng-automation`](https://github.com/coreos/fedora-coreos-releng-automation) and [`fedora-coreos-config`](https://github.com/coreos/fedora-coreos-config). You'll also want to fork both of them on GitHub.
+
+Then, from the checkout for `fedora-coreos-config` (replace `upstream` below with whichever remote name tracks `coreos/`):
+
+- [ ] `git fetch upstream`
+- [ ] `git checkout testing`
+- [ ] `git reset --hard upstream/testing`
+- [ ] `/path/to/fedora-coreos-releng-automation/scripts/promote-config.sh testing-devel`
+- [ ] sanity check promotion with `git show`
+- [ ] open PR against the `testing` branch on https://github.com/coreos/fedora-coreos-config
+- [ ] ideally have at least one other person check it and approve before merging
+
+# Build
+
+- [ ] Start a [pipeline build](https://jenkins-fedora-coreos.apps.ci.centos.org/job/fedora-coreos/job/fedora-coreos-fedora-coreos-pipeline/build?delay=0sec) (select `testing`, and fill in version number using the `30.YYYYMMDD.N` format)
+- [ ] Wait for the job to finish
+
+# Sanity-check the build
+
+Using the [the build browser](https://builds.coreos.fedoraproject.org/browser) for the `testing` stream:
+
+- [ ] Verify that the parent commit and version match the previous `testing` release (in the future, we'll want to integrate this check in the release job)
+- [ ] Run kola on AMI to sanity check it, e.g.:
+      `kola -p aws run --aws-ami <ami-id> --aws-region us-east-1 --parallel 10 -b fcos`
+      This will be run automatically on all builds in the future.
+
+# Sign the CHECKSUMS file for releng
+
+This is a stopgap until we do signing through fedora-messaging.
+
+- [ ] Download the `CHECKSUMS` file locally:
+      `aws s3 cp s3://fcos-builds/prod/streams/testing/builds/$VERSION/CHECKSUMS .`
+- [ ] **Confirm that the SHA256 of the `CHECKSUMS` file you just downloaded matches the one from the pipeline Jenkins log output**
+- [ ] Sign it with your key:
+      `gpg2 --output CHECKSUMS.sig --detach-sign CHECKSUMS`
+- [ ] Push your signature to the bucket:
+      `aws s3 cp --acl=public-read CHECKSUMS.sig s3://fcos-builds/prod/streams/testing/builds/$VERSION/CHECKSUMS.sig`
+
+# Signing artifacts and importing OSTree commit
+
+IMPORTANT: this is the point of no return here. Once the OSTree commit is
```

Comment:
What would you think of dividing the document into two L1 headings, `Prepare for release` and `Release`, and changing all the current headings to L2?